src/Mappers/SpecialKeys/Motion.ts

```diff
@@ -69,6 +69,9 @@ export class SpecialKeyMotion extends GenericMapper implements SpecialKeyCommon
 
         { keys: 'g g', motionGenerators: [MotionDocument.toLineOrFirst, MotionLine.firstNonBlank] },
         { keys: 'G',   motionGenerators: [MotionDocument.toLineOrLast, MotionLine.firstNonBlank] },
+        { keys: '%', motionGenerators: [
+            (args: {n : number}) => MotionDocument.toLinePercent({n:args.n})
+        ]},
```

Comment:
This will override
```ts
{ keys: '%', motionGenerators: [MotionMatchPair.matchPair] }
```
and disable that motion.

We need to determine if `n` is set and then creating the correct Motion.
Something like this:

```ts
{ keys: '%', motionGenerators: [
    (args: {n?: number}) => args.n === undefined
        ? MotionMatchPair.matchPair()
        : MotionDocument.toPercent(args)
] }
```